### PR TITLE
Added new feature: GIT Conflict task

### PR DESCRIPTION
**Pull request by ceesvanegmond**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documented?   | yes

A friend of mine @cafferata came up with this idea. Implement a new task which checks if your current working directory (repository) has any merge conflicts. It's a simple task and there's nothing special to it. I though it was a good idea, so I implemented it. Got the GIT command out of [StackOverflow](http://stackoverflow.com/questions/3065650/whats-the-simplest-way-to-git-a-list-of-conflicted-files)

What do you guys think of this idea? Maybe we can make this idea better?

### CODE REVIEW

**Comment by veewee** [comment]
I like the idea a lot! It will also result in less "Fixed merge conflicts" commits.
Don't know how we could make it better at the moment, but we can launch this one and improve it when we've got new ideas about the task.

Thanks for the PR!

File: src/GrumPHP/Task/Git/Conflict.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+namespace GrumPHP\Task\Git;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\AbstractExternalTask;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Git Blacklist Task
```
Comment (tifabien) [inline src/GrumPHP/Task/Git/Conflict.php]: Think due to a task copy paste :)

Comment (ceesvanegmond) [inline src/GrumPHP/Task/Git/Conflict.php]: @tifabien Fixed!

**Comment by veewee** [comment]
Thanks!

**Pull request merged**